include/ice_usb.h

```diff
@@ -196,8 +196,8 @@ extern const tusb_desc_device_t tud_desc_device;
 extern void (*tud_cdc_rx_cb_table[CFG_TUD_CDC])(uint8_t);
 
 void ice_usb_init(void);
-void ice_usb_sleep_ms(uint32_t ms);
```

Comment:
This was a way for the user  to easily sleep without having to manually roll a loop that calls the USB task.
if sleeping is for microseconds only, this would not lead to a timeout on the host side, so the regular sleep can be used instead.

Does this mean you would rather use `ice_usb_task()` directly?

Reply:
I did a cleanup. To make the SDK more concise and minimal. But I guess this PR is not a good place for removing existing functionality. Will restore that.

Reply:
I like the idea of providing fewer. But good point about making a separate PR.